src/views/payment-review.njk

```diff
@@ -0,0 +1,57 @@
+{% extends "layout.njk" %}
+
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set title = "Review your payment - " + pageTitleSuffix %}
+
+{% block content %}
+  <form method="post">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">Review your payment</h1>
+
+        {% set paymentRows = [] %}
+        {% for payment in paymentSummary.payments %}
+          {% set paymentRows = (
+            paymentRows.push(
+              {
+                text: payment.description,
+                classes: 'item-' + loop.index0
+              },
+              {
+                text: payment.cost,
+                classes: 'amount-to-pay-' + loop.index0,
+                format: "numeric"
+              }
+            ), paymentRows
```

Comment:
what does this value do?

Reply:
It's the second parameter in the `set`, and the value passed to the `push` function to be used as its "this" value (for building the `paymentRows` array).

Think of it as the `thisArg` parameter in the `map` method we use for arrays.